src/main/java/me/modmuss50/optifabric/patcher/LambadaRebuiler.java

```diff
@@ -1,5 +1,6 @@
 package me.modmuss50.optifabric.patcher;
 
+import me.modmuss50.optifabric.util.ASMUtils;
```

Comment:
Is this import needed?

Reply:
This happened because I moved ASMUtils to the util directory. That directory didn't exist yet before the backports and it seemed like a better place for ASMUtils to be (is also consistent with the llama branch)

Reply:
Reminds me that I didn't do that for the alpaca branch, should I add that to that PR as well for consistency?

Reply:
If you want to, the package structure is all quite improvised as it is anyway

Reply:
Alright, fair enough. I'll move it just for consistency sake, since I already did it on this branch.